src/README.md

```diff
@@ -193,6 +193,10 @@ These have been internally mapped to set volume to 0 and it's previous value.
 
 Lastly, there are two other states available, one that has already been mentioned:
```

Comment:
this line should be included in the Ready and Pause states section, probably want to tailor the language a bit (or just move the playOptions above)